packages/config/config/devices/0x0413/re_guard.json

```diff
@@ -0,0 +1,484 @@
+{
+	"manufacturer": "REHAU AG + Co",
+	"manufacturerId": "0x0413",
+	"label": "RE.GUARD",
+	"description": "Water Leakage Detector and Meter",
+	"devices": [
+		{
+			"productType": "0x0001",
+			"productId": "0x0001"
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"associations": {
+		"1": {
+			"label": "Lifeline",
+			"description": "Connection to Gateway",
+			"maxNodes": 1,
+			"isLifeline": true
+		},
+		"2": {
+			"label": "Valve Status",
+			"description": "Sends Status of Valve",
+			"maxNodes": 5
+		},
+		"3": {
+			"label": "Leakage Alarm",
+			"description": "Sends notification in Case of Leakage",
+			"maxNodes": 5
+		}
+	},
+	"paramInformation": [
+		{
+			"#": "1",
+			"label": "Operation Mode",
+			"description": "0 valve always off; 1 user absent/holiday; 2 user present; 3 valve always on",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 3,
+			"defaultValue": 2,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Valve Always Off",
+					"value": 0
+				},
+				{
+					"label": "User Absent/Holiday",
+					"value": 1
+				},
+				{
+					"label": "User Present",
+					"value": 2
+				},
+				{
+					"label": "Valve Always On",
+					"value": 3
+				}
+			]
+		},
+		{
+			"#": "4",
+			"label": "Microleakage Test (hour)",
+			"unit": "h",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 23,
+			"defaultValue": 3
+		},
+		{
+			"#": "5",
+			"label": "Microleakage Test (minute)",
+			"unit": "min",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 59,
+			"defaultValue": 0
+		},
+		{
+			"#": "6",
+			"label": "Fixed Schedule Day",
+			"description": "fixed day in bin 1111111, Su=1, Sa=2, Fr=4, Th=8, We=16, Tu=32, Mo=64, 0= auto",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 127,
+			"defaultValue": 127
+		},
```

Comment:
This one should be split into partial parameters: https://zwave-js.github.io/node-zwave-js/#/config-files/partial-parameters